upgrade_pythoncapi.py

```diff
@@ -578,15 +593,20 @@ def _parse_options(self, args):
         self.operations = self._get_operations(parser)
 
     def main(self):
-        for filename in self.walk(self.args.paths):
-            self.patch_file(filename)
+        if self.args.paths:
+            for filename in self.walk(self.args.paths):
+                self.patch_file(filename)
+
+        if self.args.download:
+            path = self.args.download
+            self.log(f"Download {PYTHONCAPI_COMPAT_H} to {path}")
+            self.get_latest_header(path)
 
         if self.pythoncapi_compat_added and not self.args.quiet:
             self.log()
             self.log(f"{INCLUDE_PYTHONCAPI_COMPAT} added: you may have "
                      f"to copy {PYTHONCAPI_COMPAT_H } to your project")
-            self.log("It can be copied from:")
-            self.log(PYTHONCAPI_COMPAT_URL)
+            self.log("Run 'python upgrade_pythoncapi.py -d <target_path>'")
```

Comment:
The message is logged even if --download is used?

I also prefer long option name in the doc:

```suggestion
            self.log("Run 'python upgrade_pythoncapi.py --download <target_path>'")
```

People tired of long options can dig into the doc to learn about short options.